domains/sailesh.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Sailesh's Personal Website",
+  "owner": {
+    "username": "saileshbro",
+    "email": "saileshbro@gmail.com"
+  },
+  "record": {
+    "CNAME": "saileshbro.github.io",
+    "URL": "https://saileshbro.github.io/saileshbro/"
```

You cannot have a cname and url redirection together this way. It should either be a cname record to your github pages link or a url redirection.

```suggestion
    "CNAME": "saileshbro.github.io"
```